include/dxc/HLSL/DxilSpanAllocator.h

```diff
@@ -61,6 +61,17 @@ class SpanAllocator {
     return Find(size, next, pos, align);
   }
 
+  // Finds the farthest position at which an element could be allocated.
+  bool FindForUnbounded(T_index &pos) {
```

Comment:
Great, but should this take an align arg, like AllocateUnbounded, and pass it along to IncPos?

Reply:
I can do that